adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/AlignmentRecordRDDFunctions.scala

```diff
@@ -712,7 +689,7 @@ class AlignmentRecordRDDFunctions(rdd: RDD[AlignmentRecord]) extends Serializabl
    *
    * @return SingleReadBuckets with primary, secondary and unmapped reads
    */
-  def adamSingleReadBuckets(): RDD[SingleReadBucket] = {
+  def groupReads(): RDD[SingleReadBucket] = {
```

Comment:
I went with the doc comment, ```Groups all reads by record group and read name```.  I don't think the word Bucket belongs in software.